.zenodo.json

```diff
@@ -56,27 +28,44 @@
       "orcid": "0000-0003-1576-6757"
     },
     {
-      "name": "Stephan, Jan",
-      "affiliation": "CASUS, Helmholtz-Zentrum Dresden-Rossendorf",
-      "orcid": "0000-0001-7839-4386"
+      "name": "Tascon, Andres Rios",
+      "affiliation": "Princeton University"
```

Comment:
@ariostas I hope this entry is correct.

Reply:
My last name is "Rios Tascon", but it's always confusing so it's not a big deal. Thank you for adding me!